src/core/filesystem.cc

```diff
@@ -510,6 +538,21 @@ GCSFileSystem::ReadTextFile(const std::string& path, std::string* contents)
   return Status::Success;
 }
 
+Status
+GCSFileSystem::DownloadFileFolder(
+    const std::string& path, std::string* local_path)
+{
+  *local_path = path;
```

Comment:
Some comment here as to why we don't need to download because using TF built-in support for accessing GCS.